src/exchange/translations/Messages_fr_FR.xml

```diff
@@ -358,12 +358,10 @@
    <translation id="exchange_update_billing_expiration_date_label" qtlid="30406">Expiration</translation>
    <translation id="exchange_update_billing_current_periode_label" qtlid="190979">Valeur actuelle</translation>
    <translation id="exchange_update_billing_periode_label" qtlid="168172">Renouvellement</translation>
-   <translation id="exchange_update_billing_header_1Y" qtlid="60390">Annuel</translation>
-   <translation id="exchange_update_billing_header_1M" qtlid="60378">Mensuel</translation>
+   <translation id="exchange_update_billing_header_1Y">Annuelle</translation>
```

Comment:
Wouldn't `Renouvellement annuel` or `Période annuelle` work also ? To ease the work of translators

Reply:
This is for the table headers in the renew modal and I'm afraid a longer text would make the table overflow